config.json

```diff
@@ -306,9 +306,9 @@
           }
         },
         "buildArch": {
```

Comment:
Agreed. That would be a bigger change though, because we would need to change build definitions and CI's to use arch instead.